src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1517,6 +1517,15 @@ def get(cls, cmd, resource_group_name, container_app_name, auth_config_name):
 class ContainerAppPreviewClient(ContainerAppClient):
     api_version = PREVIEW_API_VERSION
 
+    @classmethod
+    def show_custom_domain_verification_id(cls, cmd):
```

Comment:
it belongs to containerappClient or env client?

Reply:
Added SubscriptionClient